ospfd/ospf_memory.h

```diff
@@ -52,5 +52,7 @@ DECLARE_MTYPE(OSPF_IF_PARAMS)
 DECLARE_MTYPE(OSPF_MESSAGE)
 DECLARE_MTYPE(OSPF_MPLS_TE)
 DECLARE_MTYPE(OSPF_PCE_PARAMS)
+DECLARE_MTYPE(OSPF_EXTERNAL)
+DECLARE_MTYPE(OSPF_REDISTRIBUTE)
```

Comment:
you can put these in `ospf_zebra.c` and use `DECLARE_MTYPE_STATIC`

(the daemon/daemon_memory.[ch] files are a cleanup work item; mtypes should be moved into the file that uses them as appropriate)